traffic_ops/traffic_ops_golang/profile/profiles.go

```diff
@@ -439,9 +439,9 @@ func Read(w http.ResponseWriter, r *http.Request) {
 	profileInterfaces := []interface{}{}
 
 	for _, p := range profileList {
-		// Attach Parameters if the 'param' parameter is sent
-		if _, ok := inf.Params["param"]; ok {
-			p.Parameters, err = ReadParameters(inf.Tx, inf.User, p.ID, inf.User.Can("PARAMETER-SECURE:READ"))
+		// Attach Parameters if the 'id' parameter is sent
+		if _, ok := inf.Params["id"]; ok {
+			p.Parameters, err = ReadParameters(inf.Tx, inf.User, p.ID, inf.User.Can("PARAMETER:SECURE-READ"))
```

Comment:
Could you pls use the `tc.PermParameterSecureRead` constant instead? That will avoid any future typos like this one.